cordova-lib/src/cordova/plugin.js

```diff
@@ -157,6 +157,7 @@ module.exports = function plugin(command, targets, opts) {
 
                             if (missingVariables.length) {
                                 shell.rm('-rf', pluginInfo.dir);
+                                events.emit('verbose', 'Removing ' + pluginInfo.dir + ' due to installation failure');
                                 var msg = 'Variable(s) missing (use: --variable ' + missingVariables.join('=value --variable ') + '=value).';
```

Comment:
I think if we add the change below then we will have exactly the same message to be traced twice. The pic above with log trace sample looks good.

> I think we should also emit a message about the variables missing.(events.emit(msg))